lucet-objdump/src/main.rs

```diff
@@ -118,19 +110,10 @@ impl<'a> ArtifactSummary<'a> {
     }
 
     fn get_func_name_for_addr(&self, addr: u64) -> Option<&str> {
```

Comment:
Strictly speaking this is no longer checking the symbol being named is a function, but checking for that doesn't make sense for objects like PEs anyway - mind renaming this to just `get_symbol_name_for_addr`?

Reply:
Makes sense!